h2/config.py

```diff
@@ -77,3 +106,61 @@ def header_encoding(self, value):
         if value is True:
             raise ValueError("header_encoding cannot be True")
         self._header_encoding = value
+
+    @property
+    def validate_sent_headers(self):
```

Comment:
I'll go for `outbound` -- that's more accurate, as we validate the headers when they're outbound, not necessarily sent.